spec/services/testing_ground/cloner_spec.rb

```diff
@@ -1,53 +1,52 @@
 require 'rails_helper'
 
 RSpec.describe TestingGround::Cloner do
```

Comment:
Did you mean to comment all this out? The Cloner class still exists.

(I just realised I've been submitting single comments instead of doing a review. Sorry for the spam. 😳)